debian/keylime_agent.service

```diff
@@ -0,0 +1,26 @@
+[Unit]
```

Comment:
I know, but it requires the that the tpmfs is created by systemd. This currently does not work with cargo deb, because they only allow one systemd unit to be packaged.

Reply:
It is ugly, but it seems to do the trick. Thank you!

I've updated the PR to reflect this change.